libraries/SuplaDevice/src/supla-common/proto.h

```diff
@@ -157,8 +152,10 @@ extern char sproto_tag[SUPLA_TAG_SIZE];
 #define SUPLA_SC_CALL_REGISTER_CLIENT_RESULT 90
 #define SUPLA_SC_CALL_REGISTER_CLIENT_RESULT_B 92  // ver. >= 9
 #define SUPLA_DS_CALL_DEVICE_CHANNEL_VALUE_CHANGED 100
+#define SUPLA_DS_CALL_DEVICE_CHANNEL_VALUE_CHANGED_B 102        // ver. >= 12
```

Comment:
This version is no longer available

Reply:
Ok, I'll update it

Reply:
updated to latest supla-common from 2.3 branch. Commented out two structures which are not compiling on arduino mega

Reply:
I updated to latest 2.3 version, but mentioned by you line still exists